content/posts/readme-html-deprecated.md

```diff
@@ -0,0 +1,40 @@
+title: Stop redirecting root documentation URL to "README.html"
+date: 2024-05-27
+category: Changelog
+tags: hosting, deprecation
+authors: Manuel Kaufmann
+status: published
+image: /images/readme-html-deprecated.jpg
+image_credit: Photo by <a href="https://unsplash.com/@introspectivedsgn?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Erik Mclean</a> on <a href="https://unsplash.com/photos/black-asus-laptop-computer-showing-3-00-sxiSod0tyYQ?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+We are announcing the **deprecation of redirecting root URLs (`/`) to `/README.html`**
```

Comment:
Is this what we're doing? I thought we were serving `README.html` as the index at `/`?

Reply:
Actually, it seems that it redirects all the index URLs (?), not just root. See santos' comment in https://github.com/readthedocs/readthedocs.org/pull/11348#discussion_r1612041026

Reply:
yep, this applies to all index pages.

Reply:
Do you have a suggestion for a better wording here?

Reply:
just use `index` instead of `root` :thinking: